### DISCUSSION TITLE
zulu scsi boot cdrom "iso"

### DISCUSSION THREAD

**Discussion by Diegorobo** [ZuluSCSI/ZuluSCSI-firmware#239] [General]
hello all how can i boot from cdrom iso image? I tried different ways unfortunately it always starts in hd .. can you give me a suggestion?
Thank you

**Comment by aperezbios** [comment]
@Diegorobo please explain what you've actually tried, so we don't have to suggest you do things you haven't tried. Please also include your zululog.txt file so we can see what's happening when you have ISOs on your SD card.
